docs/apps/guacamole.md

```diff
@@ -1 +1,93 @@
-Placeholder page
+# Guacamole
+
+This guide will help you replicate Gilbn's tutorial to protect your Guacamole install with F2B
+
+Since DockSTARTer uses Oznu's image for Guacamole, it only generates logs inside the container itself. Following these steps will allow you to get the Guacamole container to generate a log file in `~/.config/appdata/guacamole` which you can then mount to the LetsEncrypt container so F2B can monitor it and ban malicious IPs.
+
+You can find Gilbn's tutorial [here](https://technicalramblings.com/blog/remotely-accessing-the-unraid-gui-with-guacamole-and-vnc-web-browser/). We recommend you read it over so you get a basic understanding of what you will be doing:
+
+## Configuring Guacamole
+
+1. Create a `logback.xml` file inside `~/.config/appdata/guacamole/guacamole`
+    * `touch ~/.config/appdata/guacamole/guacamole/logback.xml`
+          or
+    * `sudo nano ~/.config/appdata/guacamole/guacamole/logback.xml`
+2. Open the file with your favorite editor and place the following contents inside of it:
+
+    **NOTE: Make sure to make changes to the timezone accordingly. Check the `php-local.ini` file in `~/.config/appdata/letsencrypt/php` if you are not sure what your timezone is.**
+
+```xml
+<configuration>
+    <!-- Appender for debugging -->
+    <appender name="GUAC-DEBUG" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS, America/New_York} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <!-- Appender for debugging in a file-->
+    <appender name="GUAC-DEBUG_FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/config/logs/guacd.log</file>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS, America/New_York} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <!-- Log at DEBUG level -->
+    <root level="debug">
+        <appender-ref ref="GUAC-DEBUG"/>
+        <appender-ref ref="GUAC-DEBUG_FILE"/>
+    </root>
+</configuration>
+```
+
+1. Restart the Guacamole container so it creates `guacd.log` in `~/.config/appdata/guacamole/logs/`
+2. Modify your docker-compose.override.yml in `~/.docker/compose/docker-compose.override.yml` file to mount the new log to letsencrypt
+    1. Example:
+
+```yaml
+  letsencrypt:
+    volumes:
+      - ${DOCKERCONFDIR}/guacamole/logs/guacd.log:/var/log/guacamole
```

Comment:
This should be `- ${DOCKERCONFDIR}/guacamole/logs/:/var/log/guacamole/`
Compose does a lot better mapping folders than mapping files. As is this would result in a folder `/var/log/` containing a file `guacamole` that is mapped to `${DOCKERCONFDIR}/guacamole/logs/guacd.log`

Reply:
Yeah that was my bad, fixing it.